src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ModelStateDictionary.cs

```diff
@@ -344,12 +342,7 @@ public ModelValidationState GetFieldValidationState(string key)
             }
 
             var entries = FindKeysWithPrefix(key);
```

Comment:
minor: Like having this temporary variable for debugging. Could you make the `ValidationState` property look similar?